intro-wikipedia/step1.md

```diff
@@ -12,7 +12,7 @@ To help you get started with Materialize, we'll:
 
 (These steps are being run for you automatically)
 
-`curl -L https://downloads.mtrlz.dev/materialized-v0.3.1-x86_64-unknown-linux-gnu.tar.gz \
+`curl -L https://binaries.materialize.com/materialized-latest-x86_64-unknown-linux-gnu.tar.gz \
```

Comment:
So this is not actually right! This will download the latest unstable version of Materialize, not the latest stable release. (The actual code that runs gets it right.) Maybe we should just drop this part and say "We're installing Materialize for you automatically", or somesuch?

Reply:
right -- i couldn't think of an easy way to do the substitution in a md file.  the other option is stick with the hardcode and manually upgrade it periodically (knowing that we're probably never going to bother).  i'll change it back to the hardcode, but i'm indifferent between the two options